### PR TITLE
Fixes #611 Add heroku auto deployment button and docs

### LINKED ISSUE #611: Add heroku auto deployment button and docs
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behavior:**

**Expected behavior:**

A heroku app auto deployment button and docs for the same so that any user can work and even deploy.
**Screenshots of the issue:**

### PR DESCRIPTION
**Pull request by bhaveshAn**
**Changes proposed in this pull request**
- Added heroku auto deployment button and docs
-
-

**Screenshots (if appropriate)** 

**Link to live demo (if appropriate):** 

**Closes #611 **

@jigyasa-grover @mariobehling Please review. Thanks !!

### CODE REVIEW

**Comment by bhaveshAn** [comment]
@AnshulMalik Please check now

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: ng serve
```
Comment (AnshulMalik) [inline Procfile]: Missing new line

Comment (bhaveshAn) [reply Procfile]: Check now

File: app.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+
+  "name": "loklak_search",
+  "scripts": {
+
+  },
+  "env": {
+     "LANG": {
```
Comment (AnshulMalik) [inline app.json]: What is this variable for ?

**Comment by AnshulMalik** [comment]
Hey @bhaveshAn, I tried deploying your branch, but I get application error, it got deployed successfully though.
Are you able deploy and run it on heroku through the button?

**Comment by bhaveshAn** [comment]
@jigyasa-grover Please review

**Comment by bhaveshAn** [comment]
@mariobehling Please review

**Comment by harshit98** [comment]
@bhaveshAn There is no need of adding Heroku. You can close this PR and the issue related to it as well.

**Comment by bhaveshAn** [comment]
Okay @harshit98

**Comment by bhaveshAn** [comment]
@mariobehling Can we add heroku now in this project. Please let me know

**Pull request closed without merging**